<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка отказов оплаты для Архангельской области</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>импортировать из</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFileName"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnSelectFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="chkProcessRelevant">
     <property name="text">
      <string>обработать по подходящим</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkImportPayed">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>загружать оплаченные</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="stat">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="layoutConfirmation">
     <item>
      <widget class="QLabel" name="lblConfirmationDir">
       <property name="text">
        <string>сохранить уведомление о получении пакета в</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtConfirmationDir"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnSelectConfirmationDir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnManualConfirmation">
       <property name="text">
        <string>Сформировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblChief">
       <property name="text">
        <string>руководитель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CPersonComboBox" name="cmbChief">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="chkImportRefused">
     <property name="text">
      <string>загружать отказы</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>начать импортирование</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelNum">
       <property name="text">
        <string>всего записей в источнике:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="20" column="0">
    <widget class="QTextBrowser" name="log"/>
   </item>
   <item row="13" column="0">
    <widget class="CProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>подтверждение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtConfirmation">
       <property name="text">
        <string>б/н</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="hlDelegate">
     <item>
      <widget class="QLabel" name="lblDelegate">
       <property name="text">
        <string>уполномоченное лицо</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CPersonComboBox" name="cmbDelegate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkOnlyCurrentAccount">
     <property name="text">
      <string>только текущий счёт</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkImportInsurance">
     <property name="text">
      <string>импорт страховой принадлжености</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
